SetCommonTools.cmd

```diff
@@ -0,0 +1,6 @@
+@echo off
```

Comment:
This is being done in a separate file because there are two locations in our internal code where we blindly use `%VS140COMNTOOLS%` and my intention is to have those locations call this script, too.